src/rct1/S4Importer.cpp

```diff
@@ -996,6 +996,12 @@ void S4Importer::ImportParkFlags()
     {
         gCheatsUnlockAllPrices = true;
     }
+    // RCT2 uses two flags for no money (for cheat detection). RCT1 used only one. 
+    // Copy its value to make no money scenarios such as Arid Heights work properly.
+    if (_s4.park_flags & PARK_FLAGS_NO_MONEY)
```

Comment:
I think it would be good to have an RCT1_PARK_FLAG enum. I believe they are almost all the same as RCT2, but it would be nice for consistency.